src/kata5.reachDestination.js

```diff
@@ -1,5 +1,5 @@
-const reachDestination = (distance, speed) => {
-
+const reachDestination = (d, s) => {
```

Comment:
Good and succinct implementation. Ideally, we call our variables more than single vowels, like `distance` instead of `d` and `speed` intead of `s`.